### PR TITLE
Add support for the DeathAdder 3.5G Black

### LINKED ISSUE #427: Razer DeathAdder black edition (1532:0029) support
I would love for my mouse to be supported by openrazer! Here is the output from Ubuntu. I'll try to find a Windows system (or setup a VM) where I can gather screenshots of the Synapse options.

```
$ lsusb -d 1532:0029 -v

Bus 003 Device 005: ID 1532:0029 Razer USA, Ltd 
Couldn't open device, some information will be missing
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 (Defined at Interface level)
  bDeviceSubClass         0 
  bDeviceProtocol         0 
  bMaxPacketSize0         8
  idVendor           0x1532 Razer USA, Ltd
  idProduct          0x0029 
  bcdDevice            1.00
  iManufacturer           1 
  iProduct                2 
  iSerial                 0 
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength           34
    bNumInterfaces          1
    bConfigurationValue     1
    iConfiguration          0 
    bmAttributes         0xa0
      (Bus Powered)
      Remote Wakeup
    MaxPower              100mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      2 Mouse
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      73
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1

$ ls -lah /dev/input/by-id/ | grep "Razer"
lrwxrwxrwx 1 root root   9 Oct 29 09:47 usb-Razer_Razer_DeathAdder-event-mouse -> ../event2
lrwxrwxrwx 1 root root   9 Oct 29 09:47 usb-Razer_Razer_DeathAdder-mouse -> ../mouse0

```

### PR DESCRIPTION
**Pull request by codefiles**
Closes #427

Requires #1863 and #1864. I'm not sure I followed proper etiquette by making those separate pull requests. My thinking is they address issues that may impact other devices and should have individual reviews.

### CODE REVIEW

File: driver/razermouse_driver.h [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 #define USB_DEVICE_ID_RAZER_OROCHI_2011 0x0013
 #define USB_DEVICE_ID_RAZER_DEATHADDER_3_5G 0x0016
+#define USB_DEVICE_ID_RAZER_DEATHADDER_3_5G_BLACK 0x0029
```
Comment (z3ntu) [inline driver/razermouse_driver.h]: please sort this list by PID, so below `USB_DEVICE_ID_RAZER_MAMBA_2012_WIRELESS`

**Comment by z3ntu** [comment]
Diff looks good apart the minor comment.

About #1864, is this a hard requirement, other does it make sense to still include support for this device while keeping also keeping this bug unresolved?

**Comment by codefiles** [comment]
The issue #1864 addresses may also impact devices that are supported. If that is the case then I would not consider it a hard requirement as this device would then function the same as those impacted yet supported devices. It has not been shown to occur with other devices but testing is needed. I could be missing something here but the underlying issue, set DPI and set poll rate DBus calls happening simultaneous during initialization and the latter failing, should not be unique to this device.

**Review by z3ntu — APPROVED** [review]
Let's merge then. LGTM, thanks for the PR!

**Pull request merged**